services/integration/src/models/constitution.models/constitution.model.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
```

Comment:
the folder containing those files should be named as just `constitution` instead of `constitution.models`

Reply:
doesn't really make it complex, but it orders which class or file corresponds

Reply:
I change the name to constitution